lib/internal/errors.js

```diff
@@ -712,9 +712,8 @@ E('ERR_HTTP2_HEADER_SINGLE_VALUE',
 E('ERR_HTTP2_INFO_STATUS_NOT_ALLOWED',
   'Informational status codes cannot be used', RangeError);
 
-// This should probably be a `TypeError`.
```

Comment:
Nit: please also remove the line break above as all other errors are directly beneath each other.